hudi-common/src/main/java/org/apache/hudi/common/table/view/RemoteHoodieTableFileSystemView.java

```diff
@@ -495,4 +501,33 @@ public Option<HoodieBaseFile> getLatestBaseFile(String partitionPath, String fil
       throw new HoodieRemoteException(e);
     }
   }
+
+  /**
+   * For remote HTTP requests, to avoid repeatedly instantiating objects.
+   */
+  private class HttpRequestCheckedFunction implements RetryHelper.CheckedFunction<Response> {
+    private String url;
+    private RequestMethod method;
+    private final int timeout;
```

Comment:
can you suffix the unit to the variable. timeoutMs or timeoutSecs

Reply:
OK, modified.